contracts/src/main.sw

```diff
@@ -108,7 +111,7 @@ struct SwapWithMaximumParams {
 
 abi Exchange {
     /// Deposit coins for later adding to liquidity pool.
-    fn deposit(gas_: u64, amount_: u64, asset_id_: b256, params: ());
+    fn deposit(gas_: u64, amount_: u64, asset_id_: b256, params: DepositParams);
```

Comment:
This should no longer be needed once a new release of the SDK is cut to include https://github.com/FuelLabs/fuels-ts/pull/134